agent/transformers/influx_db.go

```diff
@@ -0,0 +1,126 @@
+package transformers
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	"github.com/sensu/sensu-go/types"
+)
+
+// InfluxList contains a list of Influx values
+type InfluxList []Influx
+
+// Influx contains values of influx db line metric format
+type Influx struct {
+	Measurement string
+	TagSet      []Tag
+	FieldSet    []Field
+	Timestamp   int64
+}
+
+// Tag is a key value pair representing an influxdb tag set
+type Tag struct {
```

Comment:
We should probably call that `InfluxTag` or move that into a `transformers.go` file so it can shared across the transformers!

Reply:
I'm actually wondering if we couldn't use `[]*types.MetricTag{}` in `Influx` and simply avoid this struct? Thoughts?

Reply:
Yeah that probably eliminates some complexity if we use `MetricTag` instead!